web-registry/src/pages/ProjectReview/ProjectReview.tsx

```diff
@@ -86,7 +91,30 @@ export const ProjectReview: React.FC = () => {
   const editPath = `/project-pages/${projectId}`;
   const creditClassId = project?.creditClassByCreditClassId?.onChainId;
   const isVCS = !!creditClassId && isVCSCreditClass(creditClassId);
-  const metadata: Partial<VCSProjectMetadataLD> = project?.metadata;
+  const metadataRaw = project?.metadata;
+
+  console.log('metadataRaw', metadataRaw);
+
+  useEffect(() => {
+    const compactMetadata = async (): Promise<void> => {
+      const meta = await jsonld.compact(
+        { ...metadataRaw },
+        {
+          schema: 'http://schema.org/',
+          regen: 'http://regen.network/',
+          qudt: 'http://qudt.org/schema/qudt/',
+          unit: 'http://qudt.org/vocab/unit/',
+          xsd: 'http://www.w3.org/2001/XMLSchema#',
+          geojson: 'https://purl.org/geojson/vocab#',
+          'geojson:coordinates': { '@container': '@list' },
```

Comment:
we should dedupe this call to jsonld.compact into a utility function, perhaps in metadata-graph.ts, so that we are not maintaining the context in two different places, maybe it could be named `jsonldCompact`?

Reply:
done